include/sisl/auth_manager/trf_client.hpp

```diff
@@ -26,9 +26,10 @@ class TrfClient {
 private:
     void validate_grant_path() const;
     bool grant_path_exists() const { return std::filesystem::exists(SECURITY_DYNAMIC_CONFIG(trf_client->grant_path)); }
+    // If leeway is set, this will force us to download token ahead of its expiry
     bool access_token_expired() const {
         return (std::chrono::system_clock::now() >
-                m_expiry + std::chrono::seconds(SECURITY_DYNAMIC_CONFIG(auth_manager->leeway)));
+                m_expiry - std::chrono::seconds(SECURITY_DYNAMIC_CONFIG(trf_client->trf_expiry_leeway_secs)));
```

Comment:
If the library `verify()` already covers the timestamp check, do we need to check it one more time here? I mean, should we still  need this "access_token_expired()" method?